### DISCUSSION TITLE
Question about clone

### DISCUSSION THREAD

**Discussion by Maid-Unknown** [cooperrc/computational-mechanics#139] [Q&A] [answered]
Dear professor.

I try to use the terminal to clone the GitHub. but I cannot make it. I don't understand why. 

I just want to know, Is it a must, or is it just a choice

I used the rocket bottom to copy everything into the notebook right now.

**Comment by cooperrc** [comment, marked as answer]
Hmm, it looks like you correctly cloned the repo, so you should be able to run

```bash
git clone https://github.com/Maid-Unknown/computational-mechanics.git
```

and have your own copy of the course materials. 

There might be an issue that the `computational-mechanics` folder already exists, so you could do this instead, 

```bash
git clone https://github.com/Maid-Unknown/computational-mechanics.git maid_comp_mech
```

it will create a folder called `maid_comp_mech` with the course material on the Jupyter server. 
This **is not** a requirement of the course. I only ask that you make a clone of the current course materials (its like buying the textbook).
